rpc/block_test.go

```diff
@@ -163,11 +173,11 @@ func TestBlockWithTxHashes(t *testing.T) {
 		if err != test.ExpectedError {
 			t.Fatal("BlockWithTxHashes match the expected error:", err)
 		}
-		switch resultBlock := result.(type) {
-		case Block:
-			block, ok := result.(*Block)
+		switch result.(type) {
+		case BlockTxHashes:
```

Comment:
Also, I don't think this case is actually being triggered for the mock test

Reply:
The reason is the method returns a pointer for BlockTxHashes, and a non-pointer for PendingTxHashes